### PR TITLE
Added resources to the readme

### PR DESCRIPTION
**Pull request by tanayvarshney**
# Description

Added resources (mainly links to documentation) to the Readme

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

Updated Readme

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
 
-More Information / System Architecture:
-
-- [GTC 2020 Talk](https://developer.nvidia.com/gtc/2020/video/s21671)
-
+Resources:
+- [Documentation](https://nvidia.github.io/Torch-TensorRT/)
+- [Torch TensorRT Explained in 2 minutes!](https://www.youtube.com/watch?v=TU5BMU6iYZ0&ab_channel=NVIDIADeveloper)
+- [Comprehensive Discusion (GTC Event w.PyTorch)](https://developer.nvidia.com/gtc/2020/video/s21671)
```
[the remark this answers is not in the document]
Comment (tanayvarshney) [reply README.md:12]: It is better if we have only 1 long session here, I can swap out the 2020 session with the 2021 session.

[the remark this answers is not in the document]
Comment (tanayvarshney) [reply README.md]: Swapped out the sessions

**Review by narendasan — APPROVED** [review]
LGTM

**Comment by narendasan** [comment]
@tanayvarshney You need to sign your commits before we can merge

**Comment by tanayvarshney** [comment]
Done!

**Pull request merged**